doc/zephyr/conf.py

```diff
@@ -192,8 +192,7 @@
 
```

Comment:
In this file, you're not removing common.css (and also not the ncs.js file, so the JavaScript stuff works) - probably that's why the doc set switcher doesn't show up right.

Reply:
fixed